tests/sdk/v2/driver/data/get_wf_logs_response/messages.py

```diff
@@ -0,0 +1,1869 @@
+# flake8: noqa
```

Comment:
I made a similar comment on Benjamin's PR.

Do we need to compare directly against all these messages? or can we compare on properties of the messages and look for sentinal values.

Having so many log lines will be annoying to update.

Reply:
I wasn't entirely happy with my approach either. I'll see how property-based assertions would play out for is.

Reply:
71593f0 WDYT

Reply:
I think thats nicer to work with in the future 👍